data/ecosystems/e/everscale.toml

```diff
@@ -2079,9 +2076,6 @@ url = "https://github.com/tonsoft-org/alpha.wallet"
 url = "https://github.com/unboxedtype/light"
 tags = ["Developer Tools", "Language"]
 
-[[repo]]
-url = "https://github.com/varnav/freeton-node-docker"
```

Comment:
apparently, you didn't get the point, let me explain that this repository markup is related to the blockchain ecosystem, and the goal is not to collect an up-to-date list - there are curated lists for this, the point of this markup is to receive commits from the start of the project and build statistics on them